Block/Adminhtml/System/Config/Fieldset/Payment.php

```diff
@@ -0,0 +1,93 @@
+<?php
+namespace Omise\Payment\Block\Adminhtml\System\Config\Fieldset;
+
+/**
+ * Fieldset renderer for Omise solution
+ */
+class Payment extends \Magento\Config\Block\System\Config\Form\Fieldset
+{
+    /**
+     * @param \Magento\Backend\Block\Context $context
+     * @param \Magento\Backend\Model\Auth\Session $authSession
+     * @param \Magento\Framework\View\Helper\Js $jsHelper
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Backend\Block\Context $context,
+        \Magento\Backend\Model\Auth\Session $authSession,
+        \Magento\Framework\View\Helper\Js $jsHelper,
+        array $data = []
+    ) {
+        parent::__construct($context, $authSession, $jsHelper, $data);
```

Comment:
@jacekstan if we removed that `$_backendConfig` then I think we don't need this `__construct()` method here anymore?

Reply:
since we removed dependency injection it was needed in other payment methods, than yes..we don't need it..